tests/Feature/Api/UserInfo/UserInfoApiTest.php

```diff
@@ -6,6 +6,7 @@
 use Hedonist\Entities\User\UserInfo;
 use Illuminate\Foundation\Testing\RefreshDatabase;
 use Illuminate\Http\UploadedFile;
+use Illuminate\Support\Facades\Storage;
 use Tests\Feature\Api\ApiTestCase;
```

Comment:
Add delete avatar test method

Reply:
Test write, when we decide how to delete the file by url